lib/forwardanalyzer.cpp

```diff
@@ -488,7 +488,7 @@ struct ForwardTraversal {
             if (allAnalysis.isIncremental())
                 return Break(Analyzer::Terminate::Bail);
         } else if (allAnalysis.isModified()) {
-            std::vector<ForwardTraversal> ftv = tryForkScope(endBlock, allAnalysis.isModified());
+            std::vector<ForwardTraversal> ftv = tryForkScope(endBlock, /*isModified*/ true);
```

Comment:
I think this is a FP. Yes its correct that it is always true, but it makes the code less clear about why it is being passed true.

Reply:
What if `isModified()` is expensive to compute?

Reply:
I would probably assign it to a variable and reuse it, but thats not the case here.